test/cql-pytest/cassandra_tests/validation/operations/select_order_by_test.py

```diff
@@ -609,23 +608,21 @@ def testAllowSkippingEqualityAndSingleValueInRestrictedClusteringColumns(cql, te
                    [0, 0, 2, 2],
                    [0, 0, 1, 1])
 
-        errorMsg = "Order by currently only supports the ordering of columns following their declared order in the PRIMARY KEY"
+        assert_invalid(cql, table, "SELECT * FROM %s WHERE a=? AND b<? ORDER BY c DESC", 0, 1)
```

Comment:
Removing the message check is not ideal because it means that if any time in the future such a query will fail for a **different** reason, the test will still pass. What I usually try to do is to pick some phrase or word which is in common to Cassandra's and Scylla's error message. But if there's no good or easy way to do this, I guess what you did here (ignoring the message) is also ok. There are many other tests that do this.

Reply:
I added checking the message back. The pattern is now `"rder by"`, because that's the largest common part between Scylla and Cassandra.

Theoretically there could be a regex that accepts all possible error message alternatives, but the problem is that `assert_invalid_message` uses `re.escape` which prevents us from passing a true regex there.